L1Trigger/Configuration/python/customiseUtils.py

```diff
@@ -142,3 +142,10 @@ def L1TGtStage2ComparisonRAWvsEMU(process):
     process.l1tgtstage2comparison = cms.Path(process.l1tComparisonGtStage2RAWvsEMU)
     process.schedule.append(process.l1tgtstage2comparison)
     return process
+
+def L1TStage2SetPrefireVetoBit(process):
+    process.load('L1Trigger.L1TGlobal.simGtExtFakeProd_cfi')
+    process.simGtExtFakeProd.tcdsRecordLabel = cms.InputTag("tcdsDigis","tcdsRecord")
+    process.l1tstage2gtext = cms.Path(process.simGtExtFakeProd)
```

Comment:
Is there a specific reason about using these two lines instead of 
```
     process.l1tstage2gtext = cms.Path(process.simGtExtUnprefireable) 
```
?

Reply:
hi Silvio, yes, I realized that this remained from the earlier version.  After the latest changes, it makes more sense to use 
```process.l1tstage2gtext = cms.Path(process.simGtExtUnprefireable)``` in the L1T custumization. 

To avoid any confusion, the L1T customization will not be used in the MiniAOD re-processing. Nevertheless, making the above change for consistency reasons.

Reply:
@silviodonato